_includes/past_issues.html

```diff
@@ -0,0 +1,19 @@
+<div class="hidden">
+  <div class="row" id="past-issues">
+    <div class="past-issue-container container">
+      <div class="container">
+        <div class="row main-content blog-detail-content contributor-sidebar">
+          {% include contributor_side_nav.html %}
+          <div class="col-md-8 offset-md-1">
+            <article class="pytorch-article">
+              {% capture march %}
+              {% include march_2021.md %}
+              {% endcapture %}
+              {{ march | markdownify }}
```

Comment:
Have you thought about how this approach will work for future issues? Will we need to come in here and hardcode each markdown file? Is it possible to instead have a folder for all of the issues, loop through them, and render them here?

Reply:
This approach is used to meet the time deadline. Looping through the issues is the ideal approach and we will have to create a new story in the near future to implement that. Andres is aware of the need to change the way this is currently implemented